db/eventlog.c

```diff
@@ -84,8 +84,28 @@ static inline void free_gbl_eventlog_fname()
     gbl_eventlog_fname = NULL;
 }
 
+static void eventlog_roll_cleanup()
+{
+    if (eventlog_nkeep == 0)
+        return;
+
+    char cmd[512] = {0};
+    char *fname = comdb2_location("logs", "%s.events", thedb->envname);
+    snprintf(cmd, sizeof(cmd) - 1, 
+             "ls -1tr %s* | head -n -%d | xargs -d '\n' rm -f --",
```

Comment:
Maybe being unreasonable paranoid here, but let's check that fname is not empty?  Afraid of an unbounded delete.

Reply:
Sure. I added check that the fname ends with '.events' for more accuracy.